api/src/pcapi/routes/backoffice/accounts.py

```diff
@@ -81,27 +81,63 @@ def get_beneficiary_credit(user_id: int) -> GetBeneficiaryCreditResponseModel:
 @blueprint.backoffice_blueprint.route("public_accounts/user/<int:user_id>/history", methods=["GET"])
 @perm_utils.permission_required(perm_models.Permissions.READ_PUBLIC_ACCOUNT)
 @spectree_serialize(
-    response_model=GetUserSubscriptionHistoryResponseModel,
+    response_model=s11n.GetUserSubscriptionHistoryResponseModel,
     on_success_status=200,
     api=blueprint.api,
 )
-def get_user_subscription_history(user_id: int) -> GetUserSubscriptionHistoryResponseModel:
+def get_user_subscription_history(user_id: int) -> s11n.GetUserSubscriptionHistoryResponseModel:
     user = users_repository.get_user_by_id(user_id)
     if not user:
         raise ApiErrors(errors={"user_id": "User does not exist"})
 
     subscriptions = {}
 
     for eligibility in list(users_models.EligibilityType):
-        subscriptions[eligibility.name] = EligibilitySubscriptionHistoryModel(
+        subscriptions[eligibility.name] = s11n.EligibilitySubscriptionHistoryModel(
             subscriptionItems=[
-                SubscriptionItemModel.from_orm(method(user, eligibility)) for method in SUBSCRIPTION_ITEM_METHODS
+                s11n.SubscriptionItemModel.from_orm(method(user, eligibility)) for method in SUBSCRIPTION_ITEM_METHODS
             ],
             idCheckHistory=[
-                IdCheckItemModel.from_orm(fraud_check)
+                s11n.IdCheckItemModel.from_orm(fraud_check)
                 for fraud_check in user.beneficiaryFraudChecks
                 if fraud_check.eligibilityType == eligibility
             ],
         )
 
-    return GetUserSubscriptionHistoryResponseModel(subscriptions=subscriptions)
+    return s11n.GetUserSubscriptionHistoryResponseModel(subscriptions=subscriptions)
+
+
+@blueprint.backoffice_blueprint.route("public_accounts/user/<int:user_id>/review", methods=["POST"])
+@perm_utils.permission_required(perm_models.Permissions.REVIEW_PUBLIC_ACCOUNT)
+@spectree_serialize(
+    response_model=s11n.BeneficiaryReviewResponseModel,
+    on_success_status=200,
+    api=blueprint.api,
+)
+def review_public_account(
+    user_id: int, body: s11n.BeneficiaryReviewRequestModel
+) -> typing.Union[s11n.BeneficiaryReviewResponseModel, tuple[Response, int]]:
```

Comment:
(détail) je ne comprends pas le type de retour. Ça n'est pas simplement ça :  

```suggestion
) -> s11n.BeneficiaryReviewResponseModel:
```

?

Je ne vois pas comment cette fonction pourrait renvoyer un `tuple[Response, int]`.